test/test_databases/test_mysql.py

```diff
@@ -108,6 +108,11 @@ def test_truncate(self):
         self.tbl.truncate()
         self.assertEqual(self.tbl.num_rows, 0)
 
+    def test_dedup_table(self):
+
+        self.tbl.dedup_table(order_by_column_name="user_name")
```

Comment:
Does this test actually pass? IT looks like the table defined in the set up doesn't have any duplicates in it, so it should still have 3 rows? Maybe this test isn't running because the MySQL database isn't set up?